apps/web/components/ChatPanel/ChatPanel.tsx

```diff
@@ -15,7 +15,7 @@ declare global {
 let boost = null
 
 export const ChatPanel = () => {
-  const [visible, setVisible] = useState<boolean>(true)
+  const [visible, setVisible] = useState<boolean>(false)
```

Comment:
I think typescript is smart enough to type useState when it's given a default value, unless it's supposed to have more types variations
```suggestion
  const [visible, setVisible] = useState(false)
```